python/composio/client/enums/base.py

```diff
@@ -37,12 +37,15 @@ class EnumStringNotFound(ComposioSDKError):
 
     def __init__(self, value: str, enum: str, possible_values: t.List[str]) -> None:
         error_message = f"Invalid value `{value}` for enum class `{enum}`"
```

Comment:
Consider refactoring the error message construction to improve readability using a list of message parts:
```python
def __init__(self, value: str, enum: str, possible_values: t.List[str]) -> None:
    message_parts = [
        f"Invalid value `{value}` for enum class `{enum}`"
    ]
    
    matches = difflib.get_close_matches(value, possible_values, n=1)
    if matches:
        (match,) = matches
        message_parts.append(f"Did you mean {match!r}?")
    
    message_parts.append(
        "(If you have recently created a custom tool, run `composio apps update` "
        "to update local apps)"
    )
    
    super().__init__(message=" ".join(message_parts))
```